airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -142,6 +168,13 @@ public AutoCloseableIterator<AirbyteMessage> read(final JsonNode config,
         });
   }
 
+  protected List<String> getCursorFields(List<CommonField<DataType>> fields) {
```

Comment:
This method has common implementation for all other sources. Please leave the only list of allowed types inside sources.